pom.xml

```diff
@@ -22,7 +22,7 @@
     <jib-maven-plugin.version>3.4.0</jib-maven-plugin.version>
     <structured-logging.version>3.0.1</structured-logging.version>
     <api-sdk-manager-java-library.version>3.0.1</api-sdk-manager-java-library.version>
-    <private-api-sdk-java.version>4.0.70</private-api-sdk-java.version>
+    <private-api-sdk-java.version>2.0.377</private-api-sdk-java.version>
```

Comment:
This looks like the Java 8 version. The Java 17 version is 4.0.80

Reply:
This doesn't look right.